Applications/Resnet/PyTorch/main.py

```diff
@@ -16,6 +16,14 @@
 from torchvision import datasets
 from torchvision.transforms import ToTensor, transforms
 
+import sys
+import os
+import numpy as np
+from collections.abc import Iterable
+
+from recorder_v2 import _get_writer
```

Comment:
`recorder_v2` seems not to want to expose `_get_writer`.

Reply:
That's a good point. I will consider to make independent utility for this task.